src/Model/EditorGroup.re

```diff
@@ -82,6 +82,38 @@ let _getAdjacentEditor = (editor: int, reverseTabOrder: list(int)) => {
   };
 };
 
+let setActiveEditorByIndexDiff = (diff, model) => {
+  let tabs = model.reverseTabOrder;
+  let count = List.length(tabs);
+
+  if (count <= 1) {
```

Comment:
It is really the case of 0 and 1 where I don't want to do anything, so checking the count makes some sense, and this felt like the best way to do that. Maybe I don't need this optimization if I make the rest of the function better.

Reply:
Oh, sorry I missed that. In that case it might be a good idea to use pattern matching. That way you can combine the 0 and 1 case:

```reason
switch (tabs) {
| [] | [_] => model
| tabs => ...
};
```

Or leaving it as-is is fine too, but there's at least some ideas for you to work with.